README.md

```diff
@@ -21,27 +21,43 @@ Atul Deshpande, Li-Fang Chu, Ron Stewart, Anthony Gitter.
 This code requires the glmnet_matlab package (http://web.stanford.edu/~hastie/glmnet_matlab/download.html).
 Unzip `glmnet_matlab.zip` in either the root directory (that contains `SCINGE_Example.m`) or the `code` subdirectory.
 
+
+## Modes of Execution
```

Comment:
Would you like me to describe the current way a user could run this inside Docker?  It is currently possible even though it isn't as elegant as the eventual solution will be.